storage/innobase/row/row0import.cc

```diff
@@ -3115,11 +3124,16 @@ static dberr_t handle_instant_metadata(dict_table_t *table,
     return err;
 
   std::unique_ptr<byte[]> page_compress_buf(new byte[get_buf_size()]);
+  std::unique_ptr<byte[], decltype(&aligned_free)> crypt_tmp_frame(
+      static_cast<byte *>(
+          aligned_malloc(physical_size, CPU_LEVEL1_DCACHE_LINESIZE)),
+      &aligned_free);
 
   if (dberr_t err= decrypt_decompress(space_crypt, space_flags,
                                       {page.get(), static_cast<size_t>
-                                       (physical_size)},
-                                      space_id, page_compress_buf.get()))
+                                      (physical_size)},
+                                      space_id, page_compress_buf.get(),
+                                      crypt_tmp_frame.get()))
```

Comment:
I think that the existing formatting was according to the default settings of GNU Emacs’ `c++-mode`.